apps/service/src/utils/json.ts

```diff
@@ -0,0 +1,28 @@
+import isPlainObject from 'lodash/isPlainObject.js';
+
+export function ensureSafeJson<Input extends object, Output = Input>(json: Input): Output {
```

Comment:
We should avoid BigInt data in the parameter instead of converting BigInt to hex-string in the JSON stringify function.

Reply:
https://github.com/search?q=repo%3Ackb-cell%2Frgbpp-sdk%20%3A%20bigint%3B&type=code shows that `bigint` has been used in `packages/ckb/src/types`.
So, maybe this `ensureSafeJson` function is necessary in the current stage？

Reply:
Yes. However, not all types will be converted to JSON strings. If we add the `ensureSafeJson` function, we should add tests to ensure it works fine.

Reply:
Sure, I can add some tests for it. Bold question: Is it possible to refactor all usages of the native `bigint` to use `BI` and hex strings instead? Would this be considered a breaking change, or is it acceptable to refactor it partially?

Reply:
Only affects JSON stringify for BigInt and I don't think it is necessary to refactor.